datajoint/autopopulate.py

```diff
@@ -118,12 +122,12 @@ def populate(self, *restrictions, suppress_errors=False, reserve_jobs=False, ord
                             jobs.complete(self.target.table_name, key)
         return error_list
 
-    def progress(self, restriction=None, display=True):
+    def progress(self, *restrictions, display=True):
```

Comment:
I agree. It looks elegant enough that we should implement it for both `progress` and `populate`.  Name collisions should be obvious to users and they will avoid them.  Does this look confusing?
```python
ScanInfo().populate(animal_id=5014, reserve_jobs=True, suppress_errors=True)
```